build.gradle

```diff
@@ -10,6 +10,7 @@ repositories {
 }
 
 dependencies {
+    implementation group:'org.xerial', name:'sqlite-jdbc', version:'3.8.11.2'
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.2'
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.2'
```

Comment:
You don't have any tests, can you remove JUnit dependencies?

Reply:
fixed